kitsune/wiki/templates/wiki/review_revision.html

```diff
@@ -16,6 +16,23 @@
       {{ _('Revision Comment: {comment}')|f(comment=revision.comment) }}
       </p>
 
+      {% if unreviewed_revision %}
+        <p>
+        <div class="unreviewed-revision"> {{ ngettext('Unreviewed Revision:', 'Unreviewed Revisions:', unreviewed_revision|length ) }} </div><br>
```

Comment:
Since this is inside the `<p>`, it should be indented one step more. Additionally, there should not be a space around the `{{ }}`. Good job with `nggettext` by the way. The is exactly how it should be used.